b2sdk/bucket.py

```diff
@@ -750,6 +750,34 @@ def hide_file(self, file_name):
         response = self.api.session.hide_file(self.id_, file_name)
         return FileVersionInfoFactory.from_api_response(response)
 
+    def copy_file(
+        self,
+        file_id,
+        new_file_name,
+        bytes_range=None,
+        metadata_directive=None,
+        content_type=None,
+        file_info=None,
+    ):
+        """
+        Creates a new file by copying from an existing file.
+
+        :param str file_id: file ID of existing file
+        :param str new_file_name: file name of the new file
+        :param tuple[int, int], None bytes_range: start and end offsets
+        :param str, None metadata_directive: `'COPY'` or `'REPLACE'`, default is `'COPY'`
+        :param str, None content_type: content_type for the new file if `'REPLACE'`, default will copy the content_type of old file
```

Comment:
Add assert to make sure it is provided in replace mode

Reply:
Already handled this in `raw_api`